docs/UserGuide.adoc

```diff
@@ -393,6 +393,19 @@ or to clear the cart after completing the purchase.
 |Clears the cart of all items.
 |===
 
+==== Move all ingredients currently in the cart to inventory
+This command allows you to move all the ingredients from the cart into your inventory. This command also
+empties your cart.
+
+- Format: `cart move ingredient`
+- Example:
+|===
+| Parameters | Result
+
+|`cart move ingredient`
```

Comment:
If we are deciding on `cart clear` instead of `cart clear ingredient`, I think this command should be changed to follow them, maybe just `cart move` or `cart stockup`

Reply:
The current implementation is `cart clear ingredient`. I did this to standardise with that command